qualtran/_infra/adjoint_test.py

```diff
@@ -162,8 +162,8 @@ def test_wire_symbol():
     (reg,) = zero.signature
     adj = Adjoint(zero)  # specifically use the Adjoint wrapper for testing
 
-    ws = zero.wire_symbol(Soquet(None, reg))
-    adj_ws = adj.wire_symbol(Soquet(None, reg.adjoint()))
+    ws = zero.wire_symbol(Soquet(BloqInstance(CNOT(), 1), reg))
+    adj_ws = adj.wire_symbol(Soquet(BloqInstance(CNOT(), 2), reg.adjoint()))
```

Comment:
this is #608 and #728; can you add a todo comment that links to #608 and either revert the change or note that the `BloqInstance`s are made up and ought to be ignored